remove the `--nw` flag once new scully version is released

Hey,

with one of the latest commits, the scully option `--nw` has been removed.
The default behaviour has been changed back to run just one and use `--watch` to explicitely start the watch mode.

https://github.com/scullyio/scully/commit/10950b5771fd4d82b5b4b2c31df7737523dad044

Probably there should be an version check to auto add `--nw` when using scully until the current version  and leaving the flag when using a newer version.

Hey, please reopen this issues, seems not to be fixed by me. The workflow fails (tested with scully version 0.0.85:

Pipeline:
https://github.com/d-koppenhagen/d-koppenhagen.de/runs/583415834?check_suite_focus=true

failing output:
```
...
chunk {4} polyfills-es5.89c31f86c1cc53c9c554.js (polyfills-es5) 130 kB [initial] [rendered]
chunk {5} styles.8cc740bfd0dbf61a8e3d.css (styles) 132 kB [initial] [rendered]
Date: 2020-04-13T18:17:43.849Z - Hash: fd66ebb2a81832f6753b - Time: 78506ms
Finished building your site.
##[error]Invalid Version: [object Object]
```

--- a/index.js
+++ b/index.js
@@ -52,6 +52,7 @@ async function run() {
     // determine the scully version
     let scullyVersion;
     if (pkgManager === 'yarn') {
+      console.log("Determine Scully version from './yarn.lock'.");
       const yarnLockRaw = await fs.readFile('./yarn.lock', 'utf8');
       const yarnLockParsed = lockfile.parse(yarnLockRaw);
       // result contains a list with e.g. "@scullyio/scully@^0.0.85" as key, so we have to find teh matching object key
@@ -59,13 +60,16 @@ async function run() {
       // use the found key to get the version from the object
       scullyVersion = yarnLockParsed.object[getScullyChildObjectKey].version;
     } else {
-      const packageJsonRaw = await fs.readFile('./package-lock.json', 'utf8');
-      const packageJsonParsed = JSON.parse(packageJsonRaw);
-      scullyVersion = packageJsonParsed.dependencies['@scullyio/scully'];
+      console.log("Determine Scully version from './package-lock.json'.");
+      const packageLockJsonRaw = await fs.readFile('./package-lock.json', 'utf8');
+      const packageLockJsonParsed = JSON.parse(packageLockJsonRaw);
+      scullyVersion = packageLockJsonParsed.dependencies['@scullyio/scully'].version;
     }
+    console.log(`Scully Version ${scullyVersion} is used`);
     
     // add the `--nw` flag if scully version is below or equal `0.0.85`
     if (semver.lte(scullyVersion, '0.0.85')) {
+      console.log(`Scully Version is less then '0.0.85', adding '--nw' flag`);
       scullyArgs = `--nw ${scullyArgs}`
     }
 

